reference/configuration.rst

```diff
@@ -47,11 +47,11 @@ Sets the maximum size to which a shard will grow before it gets split and defaul
 citus.shard_placement_policy (enum)
 $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
 
-Sets the policy to use when choosing nodes for placing newly created shards. When using the \stage command, the master needs to choose the worker nodes on which it will place the new shards. This configuration value is applicable on the master and specifies the policy to use for selecting these nodes. The supported values for this parameter are :-
+Sets the policy to use when choosing nodes for placing newly created shards. When using the \copy command, the master needs to choose the worker nodes on which it will place the new shards. This configuration value is applicable on the master and specifies the policy to use for selecting these nodes. The supported values for this parameter are :-
```

Comment:
We should probably comment out this whole parameter until Metin's changes are checked in.